static/app/views/performance/transactionSummary/transactionSpans/spanDetails/spanDetailsSearchBar.tsx

```diff
@@ -0,0 +1,36 @@
+import styled from '@emotion/styled';
+import {Location} from 'history';
+
+import _SearchBar from 'sentry/components/events/searchBar';
```

Comment:
I would avoid this and instead probably call it `StyledSearchBar = styled(SearchBar)`

Reply:
ah right, that takes us back to the default export vs named export argument
implemented your suggestion 👍